Makefile

```
@@ -39,7 +39,7 @@ test-unit: all
 	TRITON_DISABLE_LINE_INFO=0 $(PYTEST) --tb=short -s python/test/unit/language/test_line_info.py
 	# Run attention separately to avoid out of gpu memory
 	$(PYTEST) --tb=short -vs python/tutorials/06-fused-attention.py
-	$(PYTEST) --tb=short -vs python/tutorials/gluon/01-intro.py python/tutorials/gluon/02-layouts.py python/tutorials/gluon/03-async-copy.py python/tutorials/gluon/04-tma.py python/tutorials/gluon/05-wgmma.py python/tutorials/gluon/06-tcgen05.py python/tutorials/gluon/07-persistence.py python/tutorials/gluon/08-warp-specialization.py
+	$(PYTEST) --tb=short -vs python/tutorials/gluon/01-intro.py python/tutorials/gluon/02-layouts.py python/tutorials/gluon/03-async-copy.py python/tutorials/gluon/04-tma.py python/tutorials/gluon/05-wgmma.py python/tutorials/gluon/06-tcgen05.py python/tutorials/gluon/07-persistence.py python/tutorials/gluon/08-warp-specialization.py python/tutorials/gluon/09-tma-gather-scatter.py python/tutorials/gluon/10-tcgen05-copy.py python/tutorials/gluon/11-tcgen05-mma-scaled.py
```

This is getting a bit unweildy:
```suggestion
	$(PYTEST) --tb=short -vs python/tutorials/gluon/
```